wire-ios-data-model/Source/ConversationSearch/ZMFilterableConversationAdapter.swift

```diff
@@ -37,6 +37,10 @@ public struct ZMFilterableConversationAdapter: FilterableConversation {
             }
     }
 
+    public var isOneOnOne: Bool {
+        conversation.oneOnOneUser != nil
```

Comment:
why not `conversation.conversationType = .oneOnOne` ?

Reply:
I discussed this a little with @johnxnguyen in the developer channel. My understanding is:

- We should only have at most a single one on one conversation displayed per non self user.
- However it is possible there are multiple conversations with type `. oneOnOne` (duplicates or after migrating to MLS for example).

The `conversation.oneOnOneUser` helps ensure that only one of these conversations is treated at a true one on one and moved to the top of the search results.

To be safe I will add your suggestion and keep the current check:

```swift
conversation.conversationType == .oneOnOne && conversation.oneOnOneUser != nil
```

Reply:
Implemented in 95d2ad8d69956bfc4f1d56d38489ec27eea29cb9